SummerSchool/app/src/main/java/nl/rug/www/rugsummerschools/controller/MainPagerActivity.java

```diff
@@ -129,20 +126,13 @@ public void onClick(View v) {
             @Override
             public Fragment getItem(int position) {
                 switch (position) {
-                    case 0 :
-                        return mFragments[0];
-                    case 1 :
-                        return mFragments[1];
-                    case 2 :
-                        return mFragments[2];
-                    case 3 :
-                        return mFragments[3];
-                    case 4 :
-                        return mFragments[4];
-                    case 5 :
-                        return mFragments[5];
-                    default:
-                        return null;
+                    case 0 : return mFragments[0];
+                    case 1 : return mFragments[1];
+                    case 2 : return mFragments[2];
+                    case 3 : return mFragments[3];
+                    case 4 : return mFragments[4];
+                    case 5 : return mFragments[5];
```

Comment:
this seems quite duplicated with a method i've seen before.
is this ok? 
in that case, you check for some login info. 
if that's not needed here, then it seems to me that you can simplify and just return mFragments[position]. or?

Reply:
Would be better. Agree.